assets/styles/homepage.scss

```diff
@@ -64,6 +64,10 @@ p {
   padding-top: 6rem;
   text-align: center;
 
+  @media screen and (min-aspect-ratio: 10/7) {
+    background-image: url('../images/houses-wide.svg');
+  }
+
   @media (max-height: 400px) {
```

Comment:
This should be at least `435` in order to make the background disappear a bit earlier.